persistent/Database/Persist/Types/Base.hs

```diff
@@ -155,6 +173,14 @@ data EntityDef = EntityDef
     -- ^ Optional comments on the entity.
     --
     -- @since 2.10.0
+    , entitySpan :: !(Maybe Span)
+    -- ^ Source code span occupied by this entity. May be absent if it is not
+    -- known.
```

Comment:
I guess we wouldn't have this if we skip the quasiquoter entirely and write the instances ourselves. Valid